jwt/api_jwt.py

```diff
@@ -236,9 +261,9 @@ def decode(
     def _validate_claims(
         self,
         payload: dict[str, Any],
-        options: dict[str, Any],
-        audience=None,
-        issuer=None,
+        options: FullOptions,
+        audience: Iterable[str] | str | None = None,
+        issuer: Iterable[str] | str | None = None,
```

Comment:
```suggestion
        issuer: Container[str] | str | None = None,
```
see discussion in #1040